packages/shared/lib/sdk/sync.ts

```diff
@@ -457,42 +439,29 @@ export class NangoAction {
             throw new Error('There is no current activity log stream to log to');
         }
 
-        if (this.usePersistAPI) {
-            const response = await persistApi({
-                method: 'POST',
-                url: `/environment/${this.environmentId}/log`,
-                data: {
-                    activityLogId: this.activityLogId,
-                    level: userDefinedLevel?.level ?? 'info',
-                    msg: content
-                }
-            });
-
-            if (response.status > 299) {
-                console.error(
-                    `Request to persist API (log) failed: errorCode=${response.status} response='${JSON.stringify(response.data)}'`,
-                    JSON.stringify(this, (key, value) => {
-                        if (key === 'secretKey') {
-                            return '********';
-                        }
-                        return value;
-                    })
-                );
-                throw new Error(`Cannot save log for activityLogId '${this.activityLogId}'`);
+        const response = await persistApi({
+            method: 'POST',
+            url: `/environment/${this.environmentId}/log`,
+            data: {
+                activityLogId: this.activityLogId,
+                level: userDefinedLevel?.level ?? 'info',
+                msg: content
             }
-            return;
-        }
+        });
 
-        await createActivityLogMessage(
-            {
-                level: userDefinedLevel?.level ?? 'info',
-                environment_id: this.environmentId as number,
-                activity_log_id: this.activityLogId as number,
-                content,
-                timestamp: Date.now()
-            },
-            false
-        );
+        if (response.status > 299) {
+            console.error(
+                `Request to persist API (log) failed: errorCode=${response.status} response='${JSON.stringify(response.data)}'`,
+                JSON.stringify(this, (key, value) => {
+                    if (key === 'secretKey') {
+                        return '********';
+                    }
+                    return value;
+                })
```

Comment:
Maybe this logic could be put in a reusable helpers

Reply:
https://github.com/NangoHQ/nango/pull/1622/commits/3086b7f6cb4e1648a560f6ea563ea871822836f3

Reply:
nice 🙏🏻